src/app/ConfigureUtilities.jsx

```diff
@@ -0,0 +1,112 @@
+import React from 'react';
+import IconSettings from 'terra-icon/lib/icon/IconSettings';
+
+function generateItemConfig(defaultItem, items, key) {
```

Comment:
Typically we've been writing utility functions with the arrow syntax as
```
const generateItemConfig = (defaultItem, items, key) => {
    ...
}
```

Reply:
Why is that preferred?

Reply:
es6 syntax? I'm not sure, its just the pattern we've followed in other component's utils ([Table](https://github.com/cerner/terra-core/blob/master/packages/terra-table/src/SelectableUtils.js), [hookshot](https://github.com/cerner/terra-core/blob/master/packages/terra-hookshot/src/_HookshotUtils.js), [app-layout](https://github.com/cerner/terra-framework/blob/master/packages/terra-application-layout/src/utils/utilityHelpers.jsx), ect)